opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -32,6 +32,12 @@
 
 #include <fmt/format.h>
 
+using MPIComm = typename Dune::MPIHelper::MPICommunicator;
+#if DUNE_VERSION_NEWER(DUNE_COMMON, 2, 7)
+    using Communication = Dune::Communication<MPIComm>;
+#else
+    using Communication = Dune::CollectiveCommunication<MPIComm>;
+#endif
```

Comment:
Please don't do that in the global namespace. The name `Communication` is rather generic and could be used somewhere else (including 3rd party libraries). Maybe doing it in an Opm::Parallel namespace is better and would accomplish the same?